Tests/Form/Type/DoctrineORMSerializationTypeTest.php

```diff
@@ -63,7 +65,12 @@ public function testBuildForm()
         $metadataFactory = $this->getMetadataFactoryMock();
         $registry = $this->getRegistryMock();
 
-        $type = new DoctrineORMSerializationType($metadataFactory, $registry, 'form_type_test', $this->class, 'serialization_api_write');
+        if (Kernel::MAJOR_VERSION < 3) {
```

Comment:
The problem with this check is that the deprecations are not removed for sf 2.8 so the Travis build will fail... :-/